examples/static-site/build/about/index.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
```

Comment:
In  my eyes, it bloats the source repository. We could publish the built examples somewhere on http://www.metalsmith.io if we want them.